### PR TITLE
[FE-230 FE-229] block setup buttons, add missing copies

### PR DESCRIPTION
**Pull request by AnCichocka**
https://oops2000.atlassian.net/browse/SCRUM-230
https://oops2000.atlassian.net/browse/SCRUM-229

- add blocking setup buttons
- add all missing copy functionalities

@Szyntos please check whether:
1. select/unselect in awards section work correctly
2. added ! in backend schema are correct

### CODE REVIEW

**Review by Szyntos — APPROVED** [review]
well, tested, works, lgtm <3

File: backend/src/main/kotlin/backend/graphql/partialPermissions/CategoryEditionPartialPermissions.kt
Comment (AnCichocka) [inline backend/src/main/kotlin/backend/graphql/partialPermissions/CategoryEditionPartialPermissions.kt:1]: isn't it the same as backend/src/main/kotlin/backend/graphql/partialPermissions/CategoriesPartialPermissions.kt?

File: backend/src/main/kotlin/backend/graphql/permissions/AwardsPermissions.kt [hunk 1]
```diff
@@ -478,6 +478,15 @@ class AwardsPermissions {
...
         }
 
+//        if (award.bonuses.isNotEmpty()){
+//            return Permission(
+//                action = action,
+//                arguments = arguments,
+//                allow = false,
+//                reason = "Award is already in use"
+//            )
+//        }
```
Comment (AnCichocka) [inline backend/src/main/kotlin/backend/graphql/permissions/AwardsPermissions.kt:488]: why commented out

File: frontend/src/graphql/setupGroupEdit.graphql.types.ts [hunk 2]
```diff
@@ -26,7 +26,7 @@ export const SetupGroupEditDocument = gql`
     $usosId: Int!
     $weekdayId: Int!
     $teacherId: Int!
-    $userIds: [Int!]!
+    $users: UserIdsType!
```
Comment (AnCichocka) [inline frontend/src/graphql/setupGroupEdit.graphql.types.ts:29]: why not [Int!]!

File: backend/src/main/kotlin/backend/graphql/partialPermissions/CategoryEditionPartialPermissions.kt
Comment (Szyntos) [reply backend/src/main/kotlin/backend/graphql/partialPermissions/CategoryEditionPartialPermissions.kt:1]: CategoriesPartialPermissions.kt deals with permissions to Categories

CategoryEditionPartialPermissions.kt deals with permissions to CategoryEditions

To different tables, two different DataFetchers

File: frontend/src/graphql/setupGroupEdit.graphql.types.ts (hunk 2 above)
Comment (Szyntos) [reply frontend/src/graphql/setupGroupEdit.graphql.types.ts:29]: UserIdsType is defined as an object with a filed userIds: [Int!]!, so the way it was done on Front wasn't working properly

this is a remnant of my first attempt at sending arrays in the request

Yes, it should be changed, but not in this PR

**Pull request merged**